tests/parent_models.py

```diff
@@ -0,0 +1,69 @@
+"""
+Define a Parent Class of Test Cases
```

Comment:
Would it be possible to have another parent class that have most of the functionalities of setUpClass, tearDownClass, setUp, and tearDown implemented? I'm not very familiar with Python so I'll write some pseudo-code below to illustrate what I mean:

```gherkin
class UnitTest
{
    // Inherited methods
    void setUpClass(self)
    {
        app.config["TESTING"] = True
        app.config["DEBUG"] = False
        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
        app.logger.setLevel(logging.CRITICAL)
        PerformDerivedSetUpClass()
    }

    void tearDownClass(self)
    {
        db.session.close()
    }

    void setUp(self)
    {
        PerformDerivedSetUp()
        db.session.query(Inventory).delete()
        db.session.commit()
    }

    void tearDown(self)
    {
        db.session.remove()
    }

    // abstract methods, must be implemented in derived classes
    void PerformDerivedSetUpClass() = 0
    void PerformDerivedSetUp() = 0
}

class TestInventoryModel : public UnitTest
{
    void PerformDerivedSetUpClass()
    {
        Inventory.init_db(app)
    }

    void PerformDerivedSetUp()
    {
        // Stub out, do nothing
    }
}

class TestInventoryServer : public UnitTest
{
    void PerformDerivedSetUpClass()
    {
        // Stub out, do nothing
    }

    void PerformDerivedSetUp()
    {
        self.client = app.test_client()
    }
}
```

If this is going to take a lot of work to implement, then you can leave it as is

Reply:
These changes won't affect the edits you made to test_routes.py and test_models.py